src/main/java/fact/TriggerEmulation/Discriminator.java

```diff
@@ -0,0 +1,120 @@
+package fact.TriggerEmulation;
+
+import fact.Constants;
+
+/**
+ *Class with functions to discriminate the signals of a given patch (or a bunch of them).
+ * Furhtermore it contains helper functions for the discrimination e.g. conversion between DAC and mV
+ */
+public class Discriminator {
+
+    public static int default_slice = 0;
+
+    /**
+     *Discriminate the signal of a given patch
+     * @param data timeseries, array[n_pixels_pe_patch]
+     * @param threshold threshold of the discriminator in DAC
+     * @param minTimeOverThreshold minimum time the signal has to stay above the threhold
+     * @param skipFirst number of slices to ignore at the beginning of the time series
+     * @param skipLast number of slices to ignore at the end of the time series
+     */
+    public static int discriminatePatch(
+            double[] data,
+            int threshold,
+            int minTimeOverThreshold,
+            int skipFirst,
+            int skipLast
+    ) {
+        int counter = 0;
```

Comment:
maybe rename this to explain *what* is being counted here?

Reply:
this counter counts the time the signal was over threshold in units of slices. So maybe: `time_over_threshold`?